models/category.js

```diff
@@ -1,13 +1,33 @@
 const { Schema, model } = require('mongoose')
-const categories = require('~/consts/categories')
 
-const categorySchema = new Schema({
-  name: {
-    type: String,
-    enum: Object.values(categories),
-    unique: true,
-    required: true
+const categorySchema = new Schema(
+  {
+    name: {
+      type: String,
+      unique: true,
+      required: true
+    },
+    categoryIcons: {
+      type: String,
+      require: true,
+    },
+    subjectsIcon: {
```

Comment:
subjectS icon? why S?

Reply:
because in schema we got 'string', what seems like it has to be general icon for all subjects in this category, and also we got this in figma 
![image](https://user-images.githubusercontent.com/79856961/223067002-15d2c469-861f-4f8f-b65d-603697751a74.png)

So i think this naming suits better